src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeoAuthProvider.scala

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.dsde.workbench.leonardo.model
+
+import java.util.UUID
+
+import org.broadinstitute.dsde.workbench.leonardo.api.AuthorizationError
+import org.broadinstitute.dsde.workbench.model.WorkbenchUserEmail
+
+import scala.concurrent.Future
+
+sealed trait ProjectAction
+case object ListClusters extends ProjectAction
+case object CreateClusters extends ProjectAction
+
+sealed trait NotebookClusterAction
+case object GetClusterDetails extends NotebookClusterAction
+case object ConnectToCluster extends NotebookClusterAction
+case object LocalizeDataToCluster extends NotebookClusterAction
+case object DestroyCluster extends NotebookClusterAction
```

Comment:
DestroyCluster isn't used anywhere

Reply:
that's because i forgot to add it to `deleteCluster`. thanks!